packages/components/src/Markdown/Markdown.tsx

```diff
@@ -23,69 +23,69 @@ interface MarkdownProps {
   readonly basicUsage?: boolean;
 }
 
-interface HeadingRendererProps {
-  readonly level: 1 | 2 | 3 | 4 | 5 | 6;
-  readonly children: ReactNode;
-}
-
-interface BaseRendererProps {
-  children: ReactNode;
-}
-
 export function Markdown({ content, externalLink, basicUsage }: MarkdownProps) {
   const props = {
     ...(basicUsage && {
-      disallowedTypes: [
-        "paragraph",
-        "heading",
+      disallowedElements: [
+        "p",
+        "h1",
+        "h2",
+        "h3",
+        "h4",
+        "h5",
+        "h6",
         "table",
-        "list",
+        "ul",
+        "li",
         "code",
         "image",
       ],
       unwrapDisallowed: true,
     }),
   };
 
+  const Tag = basicUsage ? React.Fragment : Content;
+
   return (
-    <ReactMarkdown
-      {...props}
-      source={content}
-      linkTarget={externalLink ? "_blank" : undefined}
-      renderers={{
-        root: renderWrapper,
-        paragraph: renderParagraph,
-        emphasis: renderEmphasis,
-        strong: renderStrong,
-        heading: renderHeading,
-      }}
-    />
+    <Tag>
+      <ReactMarkdown
+        {...props}
+        linkTarget={externalLink ? "_blank" : undefined}
+        components={{
+          p: renderParagraph,
+          strong: renderStrong,
+          em: renderEmphasis,
+          h1: renderHeading(1),
```

Comment:
For reference this is the replacements that are defined.
https://github.com/remarkjs/react-markdown/blob/main/changelog.md#600---2021-04-15